test/suites/test_crud.py

```diff
@@ -0,0 +1,751 @@
+import re
+import sys
+import time
+import unittest
+import tarantool
+from .lib.tarantool_server import TarantoolServer
+from tarantool.error import CrudModuleError
+
+
+def create_server():
+    srv = TarantoolServer()
+    srv.script = 'test/suites/crud_server.lua'
+    srv.start()
+
+    return srv
+
+
+class TestSuite_Crud(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(self):
+        print(' CRUD '.center(70, '='), file=sys.stderr)
+        print('-' * 70, file=sys.stderr)
+        # Create server and extract helpful fields for tests.
+        self.srv = create_server()
+        self.host = self.srv.host
+        self.port = self.srv.args['primary']
+
+    def setUp(self):
+        time.sleep(1)
+        # Open connections to instance.
+        self.conn = tarantool.Connection(host=self.host, port=self.port, 
+                                         user='guest', password='')
+        self.conn_mesh = tarantool.MeshConnection(host=self.host, port=self.port, 
+                                         user='guest', password='')
+        self.conn_pool = tarantool.ConnectionPool([{'host':self.host, 'port':self.port}], 
+                                                     user='guest', password='')
+        # Time for vshard group configuration.
+        time.sleep(1)
+
+    crud_test_cases = {
+        'crud_insert': {
+            'success': {
+                'correct_input': {
```

Comment:
Since we're already in `success`, why not use `input`/`output`? Same for error.

Reply:
Done